src/FileCabinet/SuiteScripts/TransferOrderV2/HC_SC_ImportTOItemFulfillment.js

```diff
@@ -0,0 +1,253 @@
+                'custrecord_ns_sftp_host_key',
+                'custrecord_ns_sftp_guid',
+                'custrecord_ns_sftp_default_file_dir'
+            ]
+            
+          });
+          var sftpSearchResults = customRecordSFTPSearch.run().getRange({
+              start: 0,
+              end: 1
+          });
+ 
+          var sftpSearchResult = sftpSearchResults[0];
+        
+          var sftpUrl = sftpSearchResult.getValue({
+              name: 'custrecord_ns_sftp_server'
+          });
+
+          var sftpUserName = sftpSearchResult.getValue({
+              name: 'custrecord_ns_sftp_userid'
+          });
+
+          var sftpPort = sftpSearchResult.getValue({
+              name: 'custrecord_ns_sftp_port_no'
+          });
+
+          var hostKey = sftpSearchResult.getValue({
+              name: 'custrecord_ns_sftp_host_key'
+          });
+          
+          var sftpSecret = sftpSearchResult.getValue({
+              name: 'custrecord_ns_sftp_guid'
+          });
+
+          var sftpDirectory = sftpSearchResult.getValue({
+              name: 'custrecord_ns_sftp_default_file_dir'
+          });
+
+          sftpDirectory = sftpDirectory + 'transferorderv2';
+          sftpPort = parseInt(sftpPort);
+
+          var connection = sftp.createConnection({
+              username: sftpUserName,
+              secret: sftpSecret,
+              url: sftpUrl,
+              port: sftpPort,
+              directory: sftpDirectory,
+              hostKey: hostKey
+          });
+          log.debug("Connection established successfully with SFTP server!");
+
+          var list = connection.list({
+              path: '/oms-fulfillment/',
+              sort: sftp.Sort.DATE
+          });
+
+          for (var i=0; i<list.length; i++) {
+              if (scriptObj.getRemainingUsage() < usageThreshold) {
+                log.debug('Scheduled script has exceeded the usage unit threshold.');
+                return;
+              }
+              if (!list[i].directory) {
+                  try {
+                      var fileName = list[i].name;
+      
+                      // Download the file from the remote server
+                      var downloadedFile = connection.download({
+                          directory: '/oms-fulfillment',
+                          filename: fileName
+                      });
+                      
+                      if (downloadedFile.size > 0) {
+                          log.debug("File downloaded successfully !" + fileName);
+                          var contents = downloadedFile.getContents();
+          
+                          //Parse the JSON file
+                          var orderDataList = JSON.parse(contents);
+                          var errorList = [];
+                          
+                          for (var dataIndex = 0; dataIndex < orderDataList.length; dataIndex++) {
+                              var orderId = orderDataList[dataIndex].order_id;
```

Comment:
as discussed, the keys in TO Fulfillment Feed JSON will be updated, do the required changes here